readabilipy/extractors/extract_element.py

```diff
@@ -0,0 +1,35 @@
+from bs4 import BeautifulSoup
+from ..text_manipulation import normalise_whitespace
+from ..plain_html import unwrap_elements
+import lxml.html
+
+
+def extract_element(html, xpaths):
+    """Return the relevant element (title, date or byline) from article HTML
+        xpaths should be a list of tuples, each with the xpath and a reliability score
+    """
+
+    # Discard certain html elements while keeping contents
```

Comment:
What does this get rid of? Without this we don't use BeautifulSoup at all in this function, so I was wondering whether we need it.